frontend/src/pages/Dashboard/components/DateRangeSelect.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
```

Comment:
General stuff, I was able to some how get the date pickers stacked vertically (as we discussed). Also I'm not sure if the color on the available dates should match our other selector or use a familiar color.